src-test/src/org/openbravo/test/costing/TestCostingBase.java

```diff
@@ -80,6 +82,25 @@ public void setInitialConfiguration() {
         organization.setCurrency(OBDal.getInstance().get(Currency.class, EURO_ID));
         OBDal.getInstance().save(organization);
 
+        // Create Internal Consumption Document Type if it does not exist for this context
+        OBCriteria<DocumentType> internalConsCrit = OBDal.getInstance().createCriteria(DocumentType.class);
+        internalConsCrit.add(Restrictions.eq(DocumentType.PROPERTY_DOCUMENTCATEGORY,
+            TestCostingConstants.MAT_INT_CONSUMPTION_DOC_CAT));
+        internalConsCrit.add(
+            Restrictions.eq(DocumentType.PROPERTY_TABLE, OBDal.getInstance().get(Table.class, "800168")));
```

Comment:
It would be good to take this "800168" into a named constant to know exactly what is the table being looked for